frontend/public/locales/en/home.json

```diff
@@ -29,133 +42,100 @@
   "tabs": [
     {
       "title": "Learn",
+      "heading": "Get the most out of your retirement benefits in Canada",
       "description": [
-        "Learn about the \"three pillars\" of Canada's retirement system.",
-        "There are many factors to consider before retiring to help ensure that you can comfortably retire."
-      ],
-      "link": "/learn",
-      "linksTitle": "Top Learning Links",
-      "links": [
         {
-          "title": "Government Benefits 101",
-          "url": "test",
-          "description": "Learn about CPP/QPP and OAS"
+          "text": "There are many factors to consider before you retire."
         },
         {
-          "title": "Sources of retirement income",
-          "url": "test2",
-          "description": "Your income during retirement will typically come from three main sources"
-        },
-        {
-          "title": "Financial Support",
-          "url": "test3",
-          "description": "Learn about your different financial support options."
+          "text": "Learn about the basics of Canada's old age benefits and ways to manage your money in retirement."
         }
-      ]
-    },
-    {
-      "title": "Plan",
-      "description": [
-        "Learn about the \"three pillars\" of Canada's retirement system.",
-        "There are many factors to consider before retiring to help ensure that you can comfortably retire."
       ],
-      "link": "/plan",
+      "button": {
+        "text": "Explore Retirement",
```

Comment:
Should this be sentence-cased?

Reply:
Not according to figma, unless you want it transformed via CSS?

Reply:
I don't think figma should be considered infallible. But keeping in line with my other comment, I'll mark this as resolved.